samples/matter/lock/README.rst

```diff
@@ -536,6 +642,119 @@ After building the sample and programming it to your development kit, complete t
+
+   .. parsed-literal::
+      :class: highlight
+
+      ./chip-tool doorlock set-year-day-schedule *yearday-index* *user-index* *localtime-start* *localtime-end* *destination-id* *endpoint-id*
+
+
+   * *yearday-index* is the index of the new schedule, starting from ``1``.
+     The maximum value is defined by the :kconfig:option:`CONFIG_LOCK_MAX_YEARDAY_SCHEDULES_PER_USER` kconfig option.
+   * *user-index* is the user index defined for the user created in the previous step.
+   * *localtime-start* is the starting time in Epoch Time.
+   * *localtime-end* is the ending time in Epoch Time.
+   * *destination-id* is the device node ID.
+   * *endpoint-id* is the Matter door lock endpoint, in this sample assigned to ``1``.
+
+   Both ``localtime-start`` and ``localtime-end`` are in seconds with the local time offset based on the local timezone and DST offset on the day represented by the value.
+
+   For example, use the following command to set a ``Year-day`` schedule with index ``1`` to start on Monday, May 27, 2024, at 7:00:00 AM GMT+02:00 DST and finish on Thursday, May 30, 2024, at 7:00:00 AM GMT+02:00 DST dedicated for user with ID ``2``::
+
+   .. code-block:: console
+
+      ./chip-tool doorlock set-year-day-schedule 1 2 1716786000 1717045200 1 1
+
+#. Set the example ``Holiday`` schedule using the following command:
+
+   .. parsed-literal::
+      :class: highlight
+
+      ./chip-tool doorlock set-holiday-schedule *holiday-index* *localtime-start* *localtime-end* *operating-mode* *destination-id* *endpoint-id*
+
+   * *holiday-index* is the index of the new schedule, starting from ``1``.
+   * *localtime-start* is the starting time in Epoch Time.
+   * *localtime-end* is the ending time in Epoch Time.
+   * *operating-mode* is the operating mode described in the :ref:`matter_lock_scheduled_timed_access` section of this guide.
+   * *destination-id* is the device node ID.
+   * *endpoint-id* is the Matter door lock endpoint, in this sample assigned to ``1``.
+
+   For example, use the following command to setup a ``Holiday`` schedule with the operating mode ``Vacation`` to start on Monday, May 27, 2024, at 7:00:00 AM GMT+02:00 DST and finish on Thursday, May 30, 2024, at 7:00:00 AM GMT+02:00 DST:
+
+   .. parsed-literal::
+      :class: highlight
+
+      ./chip-tool doorlock set-holiday-schedule 1 1716786000 1717045200 1 1 1
+
+#. Read saved schedules using the following commands and providing the same arguments you used in the earlier steps:
+
+   .. parsed-literal::
+      :class: highlight
+
+      ./chip-tool doorlock get-week-day-schedule *weekday-index* *user-index* *destination-id* *endpoint-id*
+
+   .. parsed-literal::
+      :class: highlight
+
+      ./chip-tool doorlock get-year-day-schedule *yearday-index* *user-index* *destination-id* *endpoint-id*
+
+   .. parsed-literal::
+      :class: highlight
+
+      ./chip-tool doorlock get-holiday-schedule *holiday-index* *destination-id* *endpoint-id*
```

Comment:
Should we add some commends to actually test if the access works? Like opening the bolt lock in defined time slot?